src/System/Console/GetOpt/Generics.hs

```diff
@@ -270,6 +310,8 @@ combine (ParseErrors e) _ = ParseErrors e
 combine (Unset _) x = x
 combine (FieldSuccess _) (ParseErrors e) = ParseErrors e
 combine (FieldSuccess a) (FieldSuccess b) = FieldSuccess (_accumulate a b)
+combine PositionalArguments _ = PositionalArguments
```

Comment:
Is this correct? `combine (ParseErrors e) PositionalArguments = ParseErrors e`, but `combine PositionalArguments (ParseErrors e) = PositionalArguments`?